dataactbroker/README.md

```diff
@@ -291,8 +291,33 @@ Example output:
 }
 ```
 
-#### GET "/v1/submission_metadata/"
-This endpoint returns metadata for the requested endpoint.
+#### GET "/v1/revalidation\_threshold/"
+This endpoint returns the revalidation threshold for the broker application. This is the date that denotes the earliest last validated date a submission must have in order to be certifiable.
```

Comment:
`earliest last`, Is it the last available or the earliest possible?

Reply:
I was going for the earliest "last_validated" but I'll make that clearer